### DISCUSSION TITLE
macOS, Monterey

### DISCUSSION THREAD

**Discussion by JPX1** [kaiserdj/Darkorbit-client#57] [Show and tell]
Hello!
in the applications there used to be a button "to run it in lower resolution". Since the macOS Monterey does allow it. Do you know how to increase performance of the application?

**Comment by kaiserdj** [comment]
You mean this?
![image](https://user-images.githubusercontent.com/5487950/160364214-8f6c2f90-963f-4566-872c-673cf86f92b1.png)

**Reply by JPX1** [reply]
Yes, exactly! In the macOS Monterey has the button disappeared... Any thoughts to run it in low resolution and not to install old OS version :D It is effecting my game performance.
Thank you very much :)

**Reply by kaiserdj** [reply]
I don't have a mac, so I can't help you very well.
But here are some links that may help you:
-[https://discussions.apple.com/thread/253340775]( https://discussions.apple.com/thread/253340775)
-[https://apple.stackexchange.com/questions/429673/low-resolution-mode-in-macos-monterey](https://apple.stackexchange.com/questions/429673/low-resolution-mode-in-macos-monterey)
-[https://www.reddit.com/r/MacOS/comments/qh2e3u/urgent_no_open_in_low_resolution_mode_in_monterey/](https://www.reddit.com/r/MacOS/comments/qh2e3u/urgent_no_open_in_low_resolution_mode_in_monterey/)

**Reply by JPX1** [reply]
Thank you, this is helpful. Therefore I am unable to open the dark orbit app, for changes the resolution. I have tried through XCODE and terminal, but I have zero to none experience with these commands :(

**Reply by JPX1** [reply]
I found it! 👍 
I had to work through it and find how to work with these programs :D 
Thank you for your kind help :) Be great!
